Resources/doc/index.rst

```diff
@@ -9,7 +9,7 @@ Admin Bundle
 * `SonataDoctrinePhpcrAdminBundle <https://github.com/sonata-project/SonataDoctrinePhpcrAdminBundle>`_: integrates PHPCR with the core admin bundle (early stage)
 * `SonataPropelAdminBundle <https://github.com/sonata-project/SonataPropelAdminBundle>`_: integrates Propel with the core admin bundle (early stage)
 
-The demo website can be found in http://demo.sonata-project.org (``admin`` as user and password).
+The demo website can be found at http://demo.sonata-project.org.
```

Comment:
Wait… there is still a login there : http://xtunjb.demo.sonata-project.org/admin/login